mubench.reviewsite/src/Controllers/RunsController.php

```diff
@@ -214,6 +215,11 @@ function deleteRunAndRelated(Run $run, $detectorId)
             $misuse->reviews()->delete();
             $misuse->metadata()->dissociate();
             $misuse->misuse_tags()->detach();
+            if($run->experiment_id == 2){
```

Comment:
Instead of checking for Ex2, can we delete conditionally by `detector_id IS NOT NULL`?